src/envoy/http/authn/origin_authenticator.cc

```diff
@@ -31,34 +31,21 @@ OriginAuthenticator::OriginAuthenticator(FilterContext* filter_context,
     : AuthenticatorBase(filter_context), policy_(policy) {}
 
 bool OriginAuthenticator::run(Payload* payload) {
-  bool success = false;
+  bool triggered_success = false;
```

Comment:
[nit] Move these 2 lines after the first if to keep it closed to where it's used.